scripts/popup.js

```diff
@@ -388,10 +416,10 @@
     $('#yes_button').on('click', startDownload);
   }
 
-  function flashDownloadingNotification(imageCount) {
+  function flashDownloadingNotification() {
     if (ls.show_download_notification !== 'true') return;
 
-    var downloading_notification = $('<div class="success">Downloading ' + imageCount + ' image' + (imageCount > 1 ? 's' : '') + '...</div>').appendTo('#filters_container');
+    var downloading_notification = $('<div class="success">Downloading ' + ls.image_count + ' image' + (ls.imageCount > 1 ? 's' : '') + '...</div>').appendTo('#filters_container');
```

Comment:
I think you meant ls.image_count instead of ls.imageCount. I'll make some other changes and also fix this.